### PR TITLE
Integration3 t6 f2 95 edit rest

### PR DESCRIPTION
**Pull request by t1yena**
refactored update dao

### CODE REVIEW

**Review by bovornc — APPROVED** [review]
Cloned it and everything works fine, outside of the aforementioned "waking up the console" issue.

File: controllers/RestaurantController.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import RestaurantControllerI from "../interfaces/RestaurantControllerI";
  * <ul>
  *     <li>GET /api/restaurants to retrieve all the restaurant instances</li>
  *     <li>GET /api/restaurants/:rid to retrieve a particular restaurant instance</li>
- *     <li>POST /api/restaurants to create a new restaurant instance</li>
+ *     <li>POST /api/users/:uid/restaurants to create a new restaurant instance</li>
```
Comment (MaceeQi) [inline controllers/RestaurantController.ts:14]: Also mentioned in Yutong's pull request - path for creating a restaurant doesn't include users/:uid

File: mongoose/RestaurantSchema.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ import Restaurant from "../models/Restaurant";
  */
 const RestaurantSchema = new mongoose.Schema<Restaurant>({
     name: {type: String, required: true},
-    ownedBy: {type: mongoose.Schema.Types.ObjectId, ref: "UserModel"},
+    ownedBy: {type: mongoose.Schema.Types.ObjectId, ref: "UserModel", required: true},
     bannerPicture: String,
```
Comment (MaceeQi) [inline mongoose/RestaurantSchema.ts:24]: Also mentioned in Yutong's pull request - default pics for banner and profile picture?

**Review by MaceeQi — COMMENTED** [review]
Works well! Same two comments as Yutong's pull request - JSDoc and default pics for restaurant

**Review by nala-he — APPROVED** [review]
I added default picture in my branch, so no need to worry. Good to merge!

File: mongoose/RestaurantSchema.ts (hunk 2 above)
Comment (nala-he) [reply mongoose/RestaurantSchema.ts:24]: I added default picture in my branch

File: controllers/RestaurantController.ts (hunk 1 above)
Comment (nala-he) [reply controllers/RestaurantController.ts:14]: I added default picture in my branch

**Pull request merged**